src/IBM.Watson.VisualRecognition.v4/VisualRecognitionService.cs

```diff
@@ -105,16 +105,22 @@ public DetailedResponse<AnalyzeResponse> Analyze(List<string> collectionIds, Lis
 
                 if (collectionIds != null)
                 {
-                    var collectionIdsContent = new StringContent(string.Join(",", collectionIds.ToArray()), Encoding.UTF8, HttpMediaType.TEXT_PLAIN);
-                    collectionIdsContent.Headers.ContentType = null;
-                    formData.Add(collectionIdsContent, "collection_ids");
+                    foreach (string item in collectionIds)
```

Comment:
Do we not need to make this handwritten change anymore for the form params?

Reply:
Not in V4 - The service accepts both comma separated string and multiple form parts for collectionIds and features. We do still need a handwritten change for V3 though. It is being generated as multiple form parts but we need it to be sent as comma separated string. There's been activity in OpenApi to support these form params as comma separated strings in the next OpenApi release.

Reply:
Ah, good to know! Should be good then 👍